keyboards/novelkeys/nk87/keymaps/default/readme.md

```diff
@@ -1,7 +1,18 @@
 # The default keymap for NK87
 
-![Layer 0](https://i.imgur.com/nw29fvc.png)
+![Layer 0](https://i.imgur.com/RQvNaqqh.png)
 Fn Layer:
-![Layer 1](https://i.imgur.com/SyijLFt.png)
+![Layer 1](https://i.imgur.com/5mKCvRbh.png)
 
 Default layer is normal ANSI TKL
+
+This keymap uses  [RGB Matrix Lighting](https://docs.qmk.fm/#/feature_rgb_matrix)
+by default. VIA is not supporting configuration of RGB Matrix Lighting effects but there are still options to do the effects configurations:
```

Comment:
This statement is obsolete — VIA V3 (#18222) has support for RGB Matrix configuration (the builtin support is limited to setting mode/HSV/speed though, and you may need to customize the list of effects in the VIA JSON if you are not enabling all existing effects in the firmware for some reason).